### PR TITLE
IEP-403 Fixing NPE in the new component wizard

### PR DESCRIPTION
**Pull request by sigmaaa**
Fixed NPE, added combo button which allows to select project from the workspace

### CODE REVIEW

File: bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/wizard/NewComponentWizardPage.java [hunk 1]
```diff
@@ -102,13 +107,45 @@ public void modifyText(ModifyEvent e)
 		});
 		GridData gd = new GridData(GridData.FILL_HORIZONTAL);
 		componentName.setLayoutData(gd);
+
+		Label projectNameLbl = new Label(container, SWT.NONE);
+		projectNameLbl.setText(Messages.NewComponentWizardPage_ProjectNameLbl);
+		projectCombo = new Combo(container, SWT.BORDER);
```
Comment (kolipakakondal) [inline bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/wizard/NewComponentWizardPage.java:113]: We can make combo ready only

File: bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/wizard/NewComponentWizardPage.java [hunk 2]
```diff
@@ -91,7 +96,7 @@ public void createControl(Composite parent)
 		Label label = new Label(container, SWT.NONE);
 		label.setText(Messages.NewIdfComponentWizard_Component_name);
 		componentName = new Text(container, SWT.BORDER);
-		componentName.setText(""); //
+		componentName.setText(""); ////$NON-NLS-1$
```
Comment (kolipakakondal) [inline bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/wizard/NewComponentWizardPage.java:99]: Additional // is added

**Review by kolipakakondal — APPROVED** [review]
LGTM

**Review by sergcarpov — APPROVED** [review]

**Review by sergcarpov — APPROVED** [review]

**Review by sergcarpov — APPROVED** [review]

**Comment by sergcarpov** [comment]
yt

**Pull request merged**